yt_dlp/aes.py

```diff
@@ -456,10 +481,20 @@ def ghash(subkey, data):
 
     last_y = [0] * BLOCK_SIZE_BYTES
     for i in range(0, len(data), BLOCK_SIZE_BYTES):
-        block = data[i:i+BLOCK_SIZE_BYTES]
+        block = data[i : i + BLOCK_SIZE_BYTES]  # noqa: E203
         last_y = block_product(xor(last_y, block), subkey)
 
     return last_y
 
 
-__all__ = ['aes_encrypt', 'key_expansion', 'aes_ctr_decrypt', 'aes_cbc_decrypt', 'aes_gcm_decrypt_and_verify', 'aes_decrypt_text']
+__all__ = [
+    'aes_encrypt',
+    'key_expansion',
+    'aes_ctr_decrypt',
+    'aes_cbc_decrypt',
+    'aes_gcm_decrypt_and_verify',
+    'aes_decrypt_text',
+    'compat_aes_cbc_decrypt',
+    'compat_aes_gcm_decrypt_and_verify',
+    'compat_pycrypto_AES',
+]
```

Comment:
This is overkill imho, just add linebreak before 120. It can be annoying to maintain, but this is intrusive nobody, except `import` reads this.